jax/experimental/jax2tf/tests/primitives_test.py

```diff
@@ -402,17 +402,7 @@ def test_add_mul(self, harness: primitive_harness.Harness):
 
   @primitive_harness.parameterized(primitive_harness.lax_min_max)
   def test_min_max(self, harness: primitive_harness.Harness):
-    expect_tf_exceptions = False
-    dtype = harness.params["dtype"]
-
-    if dtype in [np.bool_, np.int8, np.uint16, np.uint32, np.uint64,
-                 np.complex64, np.complex128]:
-      # TODO(bchetioui): tf.math.maximum and tf.math.minimum are not defined for
```

Comment:
Nice cleanup! Some of these TODO's deserve to be moved to the categorizer, but probably not this one.